### ISSUE TITLE
Interactive popup blocking

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#2375]
Instead of having a `javascript-can-open-windows-automatically` setting, we should have some way to interactively allow blocked popups, like Otter Browser does.

It'd be nice to have some API in QtWebEngine for this though, so we don't have to reimplement the check whether it's resulting from an user action or not.

**Comment by Aerex** [comment]
I am not sure if this is the right issue to ask this but the current fix for this is to set `javascript-can-open-windows-auotmatically` to `True` to enable popups. I am using qutebrowser v1.4.1 and that configuration setting is not available.

**Comment by The-Compiler** [comment]
It was renamed to `content.javascript.can_open_tabs_automatically` in v1.0.0

**Comment by The-Compiler** [comment]
I opened [QTBUG-69768](https://bugreports.qt.io/browse/QTBUG-69768) for this - I don't think it's feasible for qutebrowser to do this before Qt has an API to do so.

**Comment by toofar** [comment]
This is this greasemonkey script https://github.com/AdguardTeam/PopupBlocker
But I just tried it for a bit and got mixed results. I tried it on the demo page they [recommend](http://code.ptcong.com/better-js-popunder-script/) and it appears to work but with [this page](http://www.dummysoftware.com/popupdummy_testpage.html) it doesn't appear to be of any help. Also the options page doesn't work at all (same on tampermonkey) and the dev build requires GM_getResource which I haven't implemented yet (but intend to).
